index.rst

```diff
@@ -1,61 +1,311 @@
+
+.. code-block:: py
+
+  await mtaos.cmd_preProcess.set_start(dataId=2020103000040)
+
+With all this considered, an Intra/Extra focus sequence with ``ComCam`` could be done like:
+
+.. code-block:: py
+
+  import asyncio
+  from lsst.ts import salobj
+  from lsst.ts.observatory.control.maintel import MTCS, ComCam
+
+  domain = salobj.Domain()
+
+  mtcs = MTCS(domain=domain)
+  comcam = ComCam(domain=domain)
+
+  asyncio.gather(mtcs.start_task, comcam.start_task)
+
+  # slew/track target
+  await mtcs.slew_icrs(ra=14., dec=-10., rot=0.)
+
+  # Select sources for processing in the background
+  select_sources_task = asyncio.create_task(
+    mtcs.rem.mtaos.cmd_selectSources.set_start(
+        ra=14.,
+        dec=-10.,
+        pa=0.,
+        type=WFSMode.ComCam
+      )
+    )
+
+  # piston camera hexapod
+  await mtcs.rem.hexapod_1.cmd_move.set_start(z=1000.)
+
+  # take first exposure
+  data_id_1 = await comcam.take_eng(exptime=30)
+
+  # piston hexapod
+  await mtcs.rem.hexapod_1.cmd_move.set_start(z=-1000.)
+
+  # tell MTAOS to start processing first image in the background
+  preproc_task_1 = asyncio.create_task(mtcs.rem.mtaos.cmd_preProcess.set_start(dataId=data_id_1))
+
+  # take second exposure
+  data_id_2 = await comcam.take_eng(exptime=30)
+
+  # tell MTAOS to start processing second image in the background
+  preproc_task_2 = asyncio.create_task(mtcs.rem.mtaos.cmd_preProcess.set_start(dataId=data_id_2))
+
+  # wait for background processes to finish
+  await asyncio.gather(select_sources_task, preproc_task_1, preproc_task_2)
+
+  # Process full array intra/extra focal images
+  await mtcs.rem.mtaos.cmd_runWEP.set_start(
+      intraId=data_id_1, extraId=data_id_2
+  )
+
+  # issue corrections to the components
+  await mtcs.rem.mtaos.cmd_issueCorrections.start()
+
+In the code above we used the already existing functionality of the `MTCS`_ and `ComCam`_ classes.
+Note that the process would be similar if we where to use ``LSSTCam`` instead of ``ComCam``, simply replacing the `ComCam`_ class by the equivalent when that becomes available.
+This means, SAL Scripts can be written for ``ComCam`` in a way that will be reusable for the main camera.
+
+.. _MTCS: https://ts-observatory-control.lsst.io/user-guide/maintel/mtcs-user-guide.html
+.. _ComCam: https://ts-observatory-control.lsst.io/user-guide/maintel/CCCamera-user-guide.html
+
+.. _user-provided-optical-aberration:
+
+User-provided optical aberration
+--------------------------------
+
+In some conditions is may be useful to allow users to specify optical aberrations to be introduced to the system, e.g., add a certain amount of comma to the system.
+The MTAOS can easily receive a list of Zernike coefficients, pass them through the Optical Feedback Control and compute the offsets and bending modes required.
+
+The command would simply receive an array of wavefront errors and apply them to the system properly registering the correction internally.
```

Comment:
Do we still need this since we will not support the Shack-Hartman wavefront sensor and high-speed CMOS camera anymore?

Reply:
This would enable us to test the OFC without involving the WEP.

Reply:
Got it. And maybe this command could be used for the LSST FAM mode. The DM service will calculate the wavefront error and send the data (contains the CCD information) to the `MTAOS` on summit to use. This means the command should have the argument to put the CCD name in additional to the wavefront error on that CCD.